src/xcode/ENA/ENA/Source/Client/HTTP Client/__tests__/HTTPClient+SubmitTests.swift

```diff
@@ -0,0 +1,212 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+@testable import ENA
+import Foundation
+import ExposureNotification
+import XCTest
+
+final class HTTPClientSubmitTests: XCTestCase {
+
+	let mockUrl = URL(staticString: "http://example.com")
+	let expectationsTimeout: TimeInterval = 2
+	let tan = "1234"
+
+	private var keys: [ENTemporaryExposureKey] {
+		let key = ENTemporaryExposureKey()
+		key.keyData = Data(bytes: [1, 2, 3], count: 3)
+		key.rollingPeriod = 1337
+		key.rollingStartNumber = 42
+		key.transmissionRiskLevel = 8
+
+		return [key]
+	}
+
+	func testSubmit_Success() {
+		// Arrange
+		let mockResponse = HTTPURLResponse(url: mockUrl, statusCode: 200, httpVersion: nil, headerFields: nil)
+		let mockURLSession = MockUrlSession(
+			// cannot be nil since this is not a a completion handler can be in (response + nil body)
+			data: Data(),
+			nextResponse: mockResponse,
+			error: nil
+		)
+		let client = HTTPClient(configuration: .fake, session: mockURLSession)
```

Comment:
why not use HTTPClient.makeWith?

Reply:
Yeah true, these unit tests were copy and pasted from the old unit test class and I did not modify them.

I will do that 👍